### DISCUSSION TITLE
Failed to ping thread devices from Mac connected to Wifi

### DISCUSSION THREAD

**Discussion by vipindas1984** [openthread/openthread#8471] [Q&A] [answered]
Hi,

I am unable to ping the thread devices(OTBR connected to eth0) from my machine(Mac connected to wifi).I am able to ping between thread devices. 

I followed one of the [previous discussions](https://github.com/openthread/ot-br-posix/issues/1220) in this forum. In some aspects that discussion helped me understand few things but I didn't completely follow how he solved the issue wrt OTBR_FORWARD_INGRESS. 
`pi@pi:~ $ sudo ip6tables -vL
Chain INPUT (policy ACCEPT 0 packets, 0 bytes)
 pkts bytes target     prot opt in     out     source               destination         

Chain FORWARD (policy ACCEPT 0 packets, 0 bytes)
 pkts bytes target     prot opt in     out     source               destination         
    0     0 OTBR_FORWARD_INGRESS  all      any    wpan0   anywhere             anywhere            

Chain OUTPUT (policy ACCEPT 0 packets, 0 bytes)
 pkts bytes target     prot opt in     out     source               destination         

Chain OTBR_FORWARD_INGRESS (1 references)
 pkts bytes target     prot opt in     out     source               destination         
    0     0 DROP       all      wpan0  any     anywhere             anywhere             PKTTYPE = unicast
    0     0 DROP       all      any    any     anywhere             anywhere             match-set otbr-ingress-deny-src src
    0     0 ACCEPT     all      any    any     anywhere             anywhere             match-set otbr-ingress-allow-dst dst
    0     0 DROP       all      any    any     anywhere             anywhere             PKTTYPE = unicast
    0     0 ACCEPT     all      any    any     anywhere             anywhere            
pi@pi:~ $`

Can someone tell me to understand on Chain OTBR_FORWARD_INGRESS connected to the issue? 

When I tried to ping thread device from my Mac, I see as below. 

`das@Vipins-MacBook-Pro ~ % ping6 fdde:ad00:beef:0:fadb:20d9:7422:2e56
ping6: UDP connect: No route to host
das@Vipins-MacBook-Pro ~ % 
das@Vipins-MacBook-Pro ~ % 
das@Vipins-MacBook-Pro ~ % ping6 fdde:ad00:beef:0:1e01:ea6e:b9c3:708b
ping6: UDP connect: No route to host
das@Vipins-MacBook-Pro ~ % `

`pi@pi:~ $ sudo ot-ctl ipaddr
fdde:ad00:beef:0:0:ff:fe00:fc11
fdde:ad00:beef:0:0:ff:fe00:fc38
fdde:ad00:beef:0:0:ff:fe00:fc10
fdde:ad00:beef:0:0:ff:fe00:4c00
fd11:22:0:0:9540:497b:5056:564e
fdde:ad00:beef:0:fadb:20d9:7422:2e56
fe80:0:0:0:64e8:d834:3403:522c
Done
pi@pi:~ $ `

`uart:~$ ot ipaddr
fdde:ad00:beef:0:0:ff:fe00:fc00
fd11:22:0:0:481f:ce44:fe33:db8d
fdde:ad00:beef:0:0:ff:fe00:e000
fdde:ad00:beef:0:1e01:ea6e:b9c3:708b
fe80:0:0:0:f818:1d1f:bea0:93c1
Done
`

Regards

Vipin Das

**Comment by jwhui** [comment, marked as answer]
The [Thread Border Router codelab](https://openthread.io/codelabs/openthread-border-router) provides an example of how to demonstrate IPv6 reachability across an OpenThread Border Router.
